app/controllers/admin/developers_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Admin::DevelopersController < Admin::AdminApplicationController
+
+  before_action :set_developer, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_admin!
```

Comment:
This line is not necessary, it already exist in Admin::AdminApplicationController.